Lab2/src/main/java/ua/lviv/IoT/lab2/manager/impl/ChemicalsWriter.java

```diff
@@ -0,0 +1,28 @@
+package ua.lviv.IoT.lab2.manager.impl;
+
+import ua.lviv.IoT.lab2.manager.IChemicalsWriter;
+import ua.lviv.IoT.lab2.model.Chemicals;
```

Comment:
Class names should be singular nouns: Chemical, Detergent, Kitchen, Toilet.

Reply:
Thank you for your comments. Also fixed it.